### ISSUE TITLE
How should PRs be handled?

### ISSUE THREAD

**Issue by shawndwells** [RedHatGov/ansible-workshop#2]
Not sure how many eyes are watching the project. Having @wtcross, @bhirsch70, @jduncan-rva and team self-ACK seems appropriate -- or you guys planning to monitor each others PRs?

**Comment by bhirsch70** [comment]
At this point, it's pretty much just JD and I, so we can self-ack.  Hopefully others will start adding their own playbooks and then we'll need to figure out how to scale.

**Comment by jduncan-rva** [comment]
agreed. i think if it's a substantive change (relative term, I know), we should get a second set of eyes as a good practice, but don't let anything rob velocity at this point.

-jduncan
